prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema

```diff
@@ -22,184 +23,189 @@ elements:
     csvFields: [{name: Patient_ID}]
 
   # In SimpleReport, lookup_id is the ID the user entered into the app.
-  - name: standard.patient_id
+  - name: patient_id
```

Comment:
For now, leave as patient_lookup_id as in master -   SimpleReport is NOT currently providing this value, so we really can't use it as our Patient PK.